__tests__/tasks/profile-picture.test.js

```diff
@@ -0,0 +1,90 @@
+const puppeteer = require('puppeteer');
+const API_BASE_URL = 'https://staging-api.realdevsquad.com';
+const { members } = require('../../mock-data/members');
+
+describe('Task Page - Assignee Profile Pic', () => {
+  let browser;
+  let page;
+  jest.setTimeout(60000);
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      headless: true,
+      ignoreHTTPSErrors: true,
+      args: ['--incognito', '--disable-web-security'],
+      devtools: false,
+    });
+    page = await browser.newPage();
+    await page.setRequestInterception(true);
+
+    page.on('request', (interceptedRequest) => {
+      const url = interceptedRequest.url();
+
+      if (url === `${API_BASE_URL}/members`) {
+        interceptedRequest.respond({
+          status: 200,
+          contentType: 'application/json',
+          headers: {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
+            'Access-Control-Allow-Headers': 'Content-Type, Authorization',
+          },
+          body: JSON.stringify(members),
+        });
+      } else {
+        interceptedRequest.continue();
+      }
+    });
+    await page.goto('http://localhost:8000/task/');
+    await page.waitForNetworkIdle();
+
+    // Click the button and select the status as Assigned
+    const buttonId = 'statusId';
+    const button = await page.$(`#${buttonId}`);
+    await button.click();
+    await page.waitForSelector('select.input:not(.notEditing)');
+    await page.select('select#status', 'ASSIGNED');
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
```

Comment:
Nit: we should keep this above first test

Reply:
Sure, will keep in mind.